src/server/saved_objects/service/lib/repository.test.js

```diff
@@ -271,7 +271,7 @@ describe('SavedObjectsRepository', () => {
         { type: 'index-pattern', id: 'two', attributes: { title: 'Test Two' } }
       ]);
 
-      expect(response).toEqual([
+      expect(response.saved_objects).toEqual([
```

Comment:
Mind putting the `saved_objects` key within the `toEquals()` rather than only checking the `saved_objects` property of the `response`?